scripts/upgradeable/transpile.sh

```diff
@@ -32,12 +32,8 @@ npx @openzeppelin/upgrade-safe-transpiler -D \
   -b "$build_info" \
   -i contracts/proxy/utils/Initializable.sol \
   -x 'contracts-exposed/**/*' \
-  -x 'contracts/proxy/**/*' \
-  -x '!contracts/proxy/Clones.sol' \
-  -x '!contracts/proxy/ERC1967/ERC1967Storage.sol' \
-  -x '!contracts/proxy/ERC1967/ERC1967Utils.sol' \
-  -x '!contracts/proxy/utils/UUPSUpgradeable.sol' \
-  -x '!contracts/proxy/beacon/IBeacon.sol' \
+  -x 'contracts/proxy/**/*Proxy*.sol' \
```

Comment:
A small nit: I would prefer to be specific about the contracts we're excluding. The counter example would be that we end up adding a contract that maches this pattern (e.g. let's say `SomeProxyAdmin`) and goes unnoticed

Reply:
We used to have it, and it looked like that https://github.com/OpenZeppelin/openzeppelin-contracts/pull/5941/commits/b5f802b3b4538965aff8106c06730dae2e9d7bec

I was worried the other way around, that we may add something like a `DiamondProxy.sol`, forget to update this, and we would end up with a transpiled version (which we don't want)